WordPress/Classes/ViewRelated/Stats/Period Stats/SiteStatsPeriodViewModel.swift

```diff
@@ -56,7 +56,7 @@ class SiteStatsPeriodViewModel: Observable {
         periodReceipt = store.query(.periods(date: lastRequestedDate, period: lastRequestedPeriod))
         store.actionDispatcher.dispatch(PeriodAction.refreshPeriodOverviewData(date: lastRequestedDate,
                                                                                period: lastRequestedPeriod,
-                                                                               forceRefresh: true))
+                                                                               forceRefresh: false))
```

Comment:
This needs to be true since it forces to update the siteId for the `statsServiceRemote`